From 369dfbf43a0064b70a774ccdd3dd1c1a09fd95ca Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Wed, 14 Dec 2022 22:23:31 +0100
Subject: [PATCH 3/4] pam: actually align the columns

In 9efb224443d819b7d64ec76cb94c8aa625a8abf2 was supposed to align
them, but for some reason I just added a second space everywhere.
---
 src/login/systemd-user.in | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/login/systemd-user.in b/src/login/systemd-user.in
index 06f7e36458..9a665bd959 100644
--- a/src/login/systemd-user.in
+++ b/src/login/systemd-user.in
@@ -4,19 +4,19 @@
 # Used by systemd --user instances.
 
 {% if ENABLE_HOMED %}
--account  sufficient pam_systemd_home.so
+-account sufficient pam_systemd_home.so
 {% endif %}
 account  sufficient pam_unix.so no_pass_expiry
-account  required pam_permit.so
+account  required   pam_permit.so
 
 {% if HAVE_SELINUX %}
-session  required pam_selinux.so close
-session  required pam_selinux.so nottys open
+session  required   pam_selinux.so close
+session  required   pam_selinux.so nottys open
 {% endif %}
-session  required pam_loginuid.so
-session  optional pam_keyinit.so force revoke
-session  required pam_namespace.so
+session  required   pam_loginuid.so
+session  optional   pam_keyinit.so force revoke
+session  required   pam_namespace.so
 {% if ENABLE_HOMED %}
--session  optional pam_systemd_home.so
+-session optional   pam_systemd_home.so
 {% endif %}
-session  optional pam_systemd.so
+session  optional   pam_systemd.so
-- 
2.38.1

